### ISSUE TITLE
Release 2.2.2 is missing from nuget.org?

### ISSUE THREAD

**Issue by AnthonyMastrean** [dotnet/aspnetcore#21578]
I was waiting on [a fix](https://github.com/dotnet/aspnetcore/blob/v2.2.2/src/Mvc/src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactory.cs#L113-L127) that was showing merged into [release 2.2.2](https://github.com/dotnet/aspnetcore/releases/tag/v2.2.2) but I can't find this version or any newer version in the 2.x family on nuget.org. I'm not ready to upgrade to 3.x yet.

https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Testing/

**Comment by javiercn** [comment]
@AnthonyMastrean thanks for contacting us.

ASP.NET Core 2.2 is out of support. See [here](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) for details. If the fix didn't make it into the latest published package, it means that it didn't happen. At this point we are have no current plans for any update to 2.2 packages since they've reached EOL.

I would suggest that you either upgrade to asp.net core 3.1 or downgrade to asp.net core 2.1 since those are both LTS versions.

**Comment by javiercn** [comment]
Tagging @dotnet/aspnet-build in case they have more info about the patching mechanics here, since I don't know why the package is not available.

**Comment by dougbu** [comment]
We have always serviced only a subset of 2.x; the only packages included in a particular release are those that require a fix or the largest meta-packages. One good way to see which 2.x releases are available is to check https://www.nuget.org/packages/Microsoft.AspNetCore.All/

But, as @javiercn said, we strongly recommend moving either forward to 3.1 or back to 2.1.

(Microsoft.AspNetCore.Mvc.Testing just didn't need to be patched in 2.2.)
